sdk/file_sdk/Common/tag.cpp

```diff
@@ -132,6 +112,45 @@ Tag::Tag(
   }
 }
 
+Tag::Tag(const Tag& tag)
+{
+  mLabelId = tag.GetLabelId();
+  mLabelName = tag.GetLabelName();
+  mLabelParentId = tag.GetLabelParentId();
+  mEnabled = tag.GetEnabled();
+  mMethod = tag.GetMethod();
+  mOwner = tag.GetOwner();
+  mSetTime = tag.GetSetTime();
+  mSiteId = tag.GetSiteId();
+
+  auto properties = tag.GetExtendedProperties();
+  for (size_t i = 0; i < properties.size(); i++) {
+    mExtendedProperties.push_back(properties[i]);
+  }

```

Comment:
just mExtendedProperties(tag.mExtendedProperties)

Reply:
doesn't compile

---
In reply to: [136936140](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136936140) [](ancestors = 136936140)

Reply:
it does

---
In reply to: [137280082](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r137280082) [](ancestors = 137280082,136936140)